diff --git a/2022/EventDataDetails.h b/2023/EventDataDetails.h
index 337d9e3..d4acc8e 100755
--- a/2022/EventDataDetails.h
+++ b/2023/EventDataDetails.h
@@ -72,4 +72,10 @@ union EventDataDetails
         uint32         m_buttonStatus;    // Bit flags specifying which buttons are being pressed
                                           // currently - see appendices
     } Buttons;
+
+    struct
+    {
+        uint8 overtakingVehicleIdx;       // Vehicle index of the vehicle overtaking
+        uint8 beingOvertakenVehicleIdx;   // Vehicle index of the vehicle being overtaken
+    } Overtake;
 };
