diff --git a/2022/MarshalZone.h b/2023/MarshalZone.h
index 184640b..715d60c 100755
--- a/2022/MarshalZone.h
+++ b/2023/MarshalZone.h
@@ -1,5 +1,5 @@
 struct MarshalZone
 {
     float  m_zoneStart;   // Fraction (0..1) of way through the lap the marshal zone starts
-    int8   m_zoneFlag;    // -1 = invalid/unknown, 0 = none, 1 = green, 2 = blue, 3 = yellow, 4 = red
-};
+    int8   m_zoneFlag;    // -1 = invalid/unknown, 0 = none, 1 = green, 2 = blue, 3 = yellow
+};
\ No newline at end of file
