diff --git a/2022/ParticipantData.h b/2023/ParticipantData.h
index 1b4519e..cb94280 100755
--- a/2022/ParticipantData.h
+++ b/2023/ParticipantData.h
@@ -10,4 +10,6 @@ struct ParticipantData
     char       m_name[48];               // Name of participant in UTF-8 format – null terminated
 					// Will be truncated with … (U+2026) if too long
     uint8      m_yourTelemetry;          // The player's UDP setting, 0 = restricted, 1 = public
+    uint8      m_showOnlineNames;        // The player's show online names setting, 0 = off, 1 = on
+    uint8      m_platform;               // 1 = Steam, 3 = PlayStation, 4 = Xbox, 6 = Origin, 255 = unknown
 };
