diff --git a/empty.h b/2023/TyreSetData.h
index e69de29..96b3a0b 100755
--- a/empty.h
+++ b/2023/TyreSetData.h
@@ -0,0 +1,12 @@
+struct TyreSetData
+{
+    uint8     m_actualTyreCompound;    // Actual tyre compound used
+    uint8     m_visualTyreCompound;    // Visual tyre compound used
+    uint8     m_wear;                  // Tyre wear (percentage)
+    uint8     m_available;             // Whether this set is currently available
+    uint8     m_recommendedSession;    // Recommended session for tyre set
+    uint8     m_lifeSpan;              // Laps left in this tyre set
+    uint8     m_usableLife;            // Max number of laps recommended for this compound
+    int16     m_lapDeltaTime;          // Lap delta time in milliseconds compared to fitted set
+    uint8     m_fitted;                // Whether the set is fitted or not
+};
